pyproject.toml

```diff
@@ -68,12 +68,16 @@ docs = [
 include = ["README.md", "LICENSE"]
 exclude = ["*_test.py"]
 
+[tool.ruff]
+line-length = 80
+
 [tool.ruff.lint]
 select = [
     "F",
     "E",
     "W291",  # whitespace at the end of the line
     "B023",  # pylint's cell-var-over-loop, closures capturing variables in loop
+    "E501",
```

Comment:
Worth adding a comment here like in the above two?

Reply:
ah, great catch, this just needs to be deleted, thanks!